doc/source/data/api/input_output.rst

```diff
@@ -105,6 +105,14 @@ Arrow
 .. automethod:: ray.data.Dataset.to_arrow_refs
     :noindex:
 
+MongoDB
+-------
+
+.. autofunction:: ray.data.read_mongo
+
+.. autofunction:: ray.data.Dataset.write_mongo
+    :noindex:
+
```

Comment:
shall we put this section after `TFRecords` at line 69 instead? Looks better if we do not mix the `read/write_mongo` within `from/to_xxx` APIs here.

Reply:
This is ordered by semantics. Starting here, these are third-party integrations rather than native ones above.

Reply:
Ok, sounds fair, no strong opinion here.